.github/workflows/main.yml

```diff
@@ -60,9 +60,16 @@ env:
   # what to build and test, see nbbuild/cluster.properties
   CLUSTER_CONFIG: 'full'
   
-  # default java distribution used by the setup-java action
+  # default java distribution used by the setup-java action across jobs
   # see https://github.com/actions/setup-java#supported-distributions
-  DEFAULT_JAVA_DISTRIBUTION: 'zulu'
+  # CI requirements:
+  #  - timely GA releases (no delays if possible)
+  #  - EA builds must be available for the upcomming release (note: some vendors skip builds)
+  #  - linux, win, mac (arm)
+  #  - reliable / no extra sauce
+  DEFAULT_JAVA_DISTRIBUTION: 'temurin'
+#  DEFAULT_JAVA_DISTRIBUTION: 'sapmachine'
+#  DEFAULT_JAVA_DISTRIBUTION: 'zulu'
```

Comment:
I checked again (before merge) just to be sure but zulu is still at build 19 of JDK 25 (https://cdn.azul.com/zulu/bin/). We need at least b22 (current is 23). The other vendors filtered out after running the [brute force availability check](https://gist.github.com/mbien/7911a1272e2bc77234cd7ffca35cdcb4) again. So we could switch to `temurin` or `sapmachine` - nice to have multiple options IMO.

Reply:
Maybe need defaults for GA and EA separately?  Interesting Temurin seems to be more timely with EA but usually slightly behind on GA?

Nitpick - upcoming, one `m`

Reply:
>  Interesting Temurin seems to be more timely with EA but usually slightly behind on GA?

yep. But this is not necessarily a problem for us since we don't require both conditions at once. We could switch to zulu before release since we don't need EA during releases. I haven't watched sapmachine so far so I don't know how it behaves there.